crates/astria-sequencer/src/app/tests_app/mempool.rs

```diff
@@ -48,23 +48,23 @@ async fn trigger_cleaning() {
     let (mut app, storage) = initialize_app_with_storage(None, vec![]).await;
     app.prepare_commit(storage.clone()).await.unwrap();
     app.commit(storage.clone()).await;
-    let sudo = get_judy_signing_key();
 
     // create tx which will cause mempool cleaning flag to be set
-    let tx_trigger = UnsignedTransaction {
-        params: TransactionParams::builder()
-            .nonce(0)
-            .chain_id("test")
-            .build(),
-        actions: vec![
+    let tx_trigger = UnsignedTransaction::new(
+        vec![
             FeeChangeAction {
                 fee_change: FeeChange::TransferBaseFee,
                 new_value: 10,
             }
             .into(),
         ],
-    }
-    .into_signed(&sudo);
+        TransactionParams::builder()
+            .nonce(0)
+            .chain_id("test")
+            .build(),
+    )
+    .expect("failed to build unsigned transaction")
```

Comment:
```suggestion
    .expect("can build unsigned transaction")
```

Reply:
same for other in this file

Reply:
As above: this does not provide info as to why you expect this to have worked. If you think it's trivially clear why this should have worked, prefer `unwrap()`.